pyLDAvis/js/ldavis.v3.0.0.js

```diff
@@ -211,8 +215,9 @@ var LDAvis = function(to_select, data_or_file_name, color1, color2) {
                 vis_state.term = "";
                 topic_off(document.getElementById(topicID + vis_state.topic));
                 var value_new = document.getElementById(topicID).value;
-                if (!isNaN(value_new) && value_new > 0) {
-                    value_new = Math.min(K, Math.max(1, value_new));
+                // older versions check for value_new !== "" too, why?
+                if (!isNaN(value_new) && value_new > startIndex - 1) {
+                    value_new = Math.min(K + startIndex - 1, Math.max(startIndex, value_new));
```

Comment:
see above, match this

Reply:
added commit 0a965059a9fca360e4e4ea212d88fd8657327bed